source/jormungandr/tests/departure_board_tests.py

```diff
@@ -1104,3 +1103,107 @@ def test_routes_schedule_with_invalid_forbidden_uri(self):
+                                          |  
+                                          v
+        (Open)                             (Close)
+        08:00             15:00             26:45  
+         X_s1 ------------ X_S2 ------------ X_S3
+
+        """
+        response = self.query_region("stop_points/X_S2/stop_schedules?from_datetime=20170103T000000"
+                                     "&data_freshness=base_schedule")
+
+        stop_schedules = response['stop_schedules']
+        assert stop_schedules
+        assert stop_schedules[0]['first_datetime']['date_time'] == "20171002T080000"
+        assert stop_schedules[0]['last_datetime']['date_time'] == "20171003T023000"
+
+        """              20170102                                                20170103
+                                              from_datetime 20170103T0300       
+                                                         |  
+                                                         v
+        (Open)                             (Close)              (Open)                             (Close)
+        08:00             15:00             26:45      03:00    08:00             15:00             26:45  
+         X_s1 ------------ X_S2 ------------ X_S3                X_s1 ------------ X_S2 ------------ X_S3
+
+        """
+        response = self.query_region("stop_points/X_S2/stop_schedules?from_datetime=20170103T0300"
+                                     "&data_freshness=base_schedule")
+
+        stop_schedules = response['stop_schedules']
+        assert stop_schedules
+        assert stop_schedules[0]['first_datetime']['date_time'] == "20171003T080000"
+        assert stop_schedules[0]['last_datetime']['date_time'] == "20171003T023000"
+
+        """ 
+                        20170103                                                  20170104    
+                                             from_datetime 20170103T0300        
+                                                        |  
+                                                        v
+        (Open)                             (Close)              (Open)                             (Close)
+        08:00             15:00             26:45      03:00    08:00             15:00             26:45  
+         X_s1 ------------ X_S2 ------------ X_S3                X_s1 ------------ X_S2 ------------ X_S3
+
+        """
+        response = self.query_region("stop_points/X_S2/stop_schedules?from_datetime=20170103T0300"
+                                     "&data_freshness=base_schedule")
+
+        stop_schedules = response['stop_schedules']
+        assert stop_schedules
+        assert stop_schedules[0]['first_datetime']['date_time'] == "20170104T080500"
+        assert stop_schedules[0]['last_datetime']['date_time'] == "20170104T024500"
+
+        """              20170104
+                               from_datetime 20170105T0030       
```

Comment:
- [ ] Should be nice to test also at 2:40 local (one way open the other closed) between 2 working days.
- [ ] Would be nice to test also that nothing breaks after the closing of the last VJ (or before opening of the first) : 20170105T040000, 20170107T000000 and 20170101T040000
- [ ] Can't we anticipate/test bad behavior having in data local closing time between 2am and 3am... Probably not nice on DST change? And probably more frequent to have those hours for open/close than for real stop_times.